draft-ietf-privacypass-rate-limit-tokens.md

```diff
@@ -922,6 +957,40 @@ struct {
 } Token;
 ~~~
 
+## Penalization of Invalid Clients and Issuers {#penalization}
+
+When an Attester encounters incorrect and potentially malicious behavior
+by either Clients or Issuers, it needs to "penalize" them. This involves
+keeping state about invalid protocol events, and determining when a
+particular Client or Issuer has exceeded a threshold for such events.
+
+Each category of invalid events can have a different threshold, and can apply
+to Clients, Issuers, or both. Different penalization events are described
+below, with suggested thresholds. This list is not exhaustive.
+
+- Client Key changing more than once over any two consecutive policy windows.
```

Comment:
Maybe worth clarifying how you identify (using the attester identity) the two different client keys are presented by the same "client".

Reply:
```suggestion
- Client Key changing more than once over any two consecutive policy windows
for a single Client, based on the Attester's view of Client identity.
```